generated/nifake/nifake_service.cpp

```diff
@@ -108,6 +108,30 @@ namespace nifake_grpc {
     }
   }
 
+  //---------------------------------------------------------------------
+  //---------------------------------------------------------------------
+  ::grpc::Status NiFakeService::BoolArrayInputFunction(::grpc::ServerContext* context, const BoolArrayInputFunctionRequest* request, BoolArrayInputFunctionResponse* response)
+  {
+    if (context->IsCancelled()) {
+      return ::grpc::Status::CANCELLED;
+    }
+    try {
+      auto vi_grpc_session = request->vi();
+      ViSession vi = session_repository_->access_session(vi_grpc_session.id(), vi_grpc_session.name());
+      ViInt32 number_of_elements = request->number_of_elements();
+      auto an_array_request = request->an_array();
+      std::vector<ViBoolean> an_array;
+      std::transform(an_array_request.begin(), an_array_request.end(), std::back_inserter(an_array), [](auto x) { return (ViBoolean)x; });
```

Comment:
@doshirohan, to convert from `bool` to `ViBoolean`, it's safer for the transform lambda to be `return x ? VI_TRUE : VI_FALSE;`